README.md

```diff
@@ -1 +1,114 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers (JS, PHP, Python, Ruby, Go, Java, .NET)
+- [ ] mysqldump
+- [ ] grafana
+- [ ] BI (Tableau, PowerBI, pick one)
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions with basic MySQL auth implementation
+- [ ] Audit logs (connections, queries)
+
+### Get web interfaces enterprise ready [APPS]
+
+- [ ] gitbase-web SSO (protocol TBD)
+- [ ] bblfsh-web SSO (protocol TBD)
+
+### Gitbase research [DR]
+
+- [ ] Support updates in backing data (indexes, etc)
+- [ ] Usage of caches for UAST and others
```

Comment:
this is already done. We are using a global object cache, languages cache and UAST cache.

Reply:
Actually this is about precomputing caches and persisting them.